Tests/JWTTests/EncodingTests.swift

```diff
@@ -56,12 +56,14 @@ final class EncodingTests: XCTestCase {
             }
         }
     }
+ 
+ */
 
     static let all = [
         ("testBase64ToBase64URL", testBase64ToBase64URL),
         ("testBase64URLToBase64", testBase64URLToBase64),
         ("testBase64DecodeIgnoresErrorForInvalidString", testBase64DecodeIgnoresErrorForInvalidString),
-        ("testBase64URLEncodeThrowsErrorForInvalidString", testBase64URLEncodeThrowsErrorForInvalidString),
-        ("testBase64URLDecodeThrowsErrorForInvalidString", testBase64URLDecodeThrowsErrorForInvalidString),
+/*        ("testBase64URLEncodeThrowsErrorForInvalidString", testBase64URLEncodeThrowsErrorForInvalidString),
```

Comment:
I'm planning on putting up a PR in the future to remove the different encoding types altogether. JWT spec requires Base64URL